net/mwan3/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mwan3
 PKG_VERSION:=2.6.3
```

Comment:
Could you please update this to PKG_VERSION 2.6.4?
Because this is a source change an not a Makefile change.